jiralib2.el

```diff
@@ -388,6 +398,22 @@ ARGS is an association list of the fields to set for the issue."
   "Change the issue type of ISSUE-ID to TYPE."
   (jiralib2-update-issue issue-id `(issuetype . ((name . ,type)))))
 
+(defun jiralib2-if-plan-issue (issue-id datestring)
```

Comment:
@MadEarl Looks to me that this is a custom function which only applies to you. Maybe it slipped into the commit by accident?

Reply:
Yes, indeed. This is only needed for my work-related customisations and slipped into this.